### PR TITLE
Feat: ReCaptcha

### PR DESCRIPTION
**Pull request by aarbanas**
## Description
In this PR we've added the ReCaptcha validation to the user registration process. 

For easier usage of prisma CLI we've introduced new scripts in package.json.

## Change summary
- recaptcha validation

## Checklist
- [x] I have performed a self-review of my code
- [x] I tested locally
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New package or package update introduced

### CODE REVIEW

**Comment by aarbanas** [comment]
@IMilja what do you think if we shall add `axios` in this PR or just use native `node fetch`?

**Comment by IMilja** [comment]
@aarbanas I prefer Axios above node fetch. Axios is more complete than node:fetch in my opinion.

**Comment by aarbanas** [comment]
> @aarbanas I prefer Axios over node fetch. Axios is more complete than node:fetch in my opinion.

Let's use axios than

File: src/users/users.service.ts [hunk 1]
```diff
@@ -23,6 +25,12 @@ export class UsersService {
   ) {}
 
   async create(createUserDto: CreateUserDto, strategy: ICreateStrategy) {
+    // Validate ReCaptcha
+    const isHuman = await this.validateReCaptcha(createUserDto.reCaptchaToken);
+    if (!isHuman) {
```
Comment (IMilja) [inline src/users/users.service.ts:30]: Should we rather throw a ForbiddenException instead? This will resolve into a 500 Internal Server Error.

**Review by IMilja — APPROVED** [review]
LGTM. Left one small comment which we can improve.

File: src/users/users.service.ts (hunk 1 above)
Comment (aarbanas) [reply src/users/users.service.ts:30]: I fully agree with you. Great catch! 🚀

**Review by IMilja — APPROVED** [review]
LGTM!

**Pull request merged**